### ISSUE TITLE
Add user interace for moving native tokens between relay and parachain

### ISSUE THREAD

**Issue by TorstenStueber** [pendulum-chain/portal#29]
As a user I want to be able to make cross-chain transfers of native relay chain tokens (DOT for Polkadot and KSM for Kusama) to our parachain (Pendulum or Amplitude).

This would be a DMP or UMP transfer.

**Comment by TorstenStueber** [comment]
@cr4pt0 can you provide some details about a DMP or UMP transaction would look like?

**Comment by cr4pt0** [comment]
relay-chain extrinsic encoded call data
0x630201000100312101000101008ceb888726d6c86b6152da55745648846301c3a20c82d9a4714ad707bceb2925010400000000070010a5d4e800000000

fococo or amplitude encoded call data
0x360001000010a5d4e8000000000000000000000001010101008ceb888726d6c86b6152da55745648846301c3a20c82d9a4714ad707bceb292500286bee00000000

**Comment by cr4pt0** [comment]
1. currencyId enum
![44577EBA-E847-4AD4-8FAF-3A7286E37E26](https://user-images.githubusercontent.com/102041955/212101890-753ac112-e2bf-404e-ac8f-283172279220.jpeg)
2. look at the prev my message:
- xcmPallet - reserveTransferAssets on relay chain (to transfer DOT/KSM/ROC from relay chain to pendulum/amplitude)
- xtokens - transfer on pendulum/amplitude chain (to move token back from our chain to relay-chain)

Simple transfer (not cross-chain) - 
pallet tokens -> extrinsic transfer (to transfer any token from to destination account on the same chain )
@prayagd

**Comment by gonzamontiel** [comment]
is this still a WIP ? What do we need to move forward ? cc @prayagd

**Comment by prayagd** [comment]
@gonzamontiel This is ready, Let me know if anything is missing or unclear.

**Comment by prayagd** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pendulum-6408a0365cda59c57d39bee5/issues/pendulum-chain/portal/29?planning-poker) with Zenhub @gonzamontiel

**Comment by gonzamontiel** [comment]
Since it is the first time we implement this, I was a bit conservative with the estimation. Ready.

**Comment by prayagd** [comment]
@gonzamontiel Would you suggest i break down this ticket? As we already discussed having smaller tickets.

**Comment by gonzamontiel** [comment]
Hmm in this case I think there's no need for that

**Comment by prayagd** [comment]
Blocked by HRMP channel between Assethub and foucoco.

**Comment by vadaynujra** [comment]
@prayagd why is this blocked by the HRMP between Foucoco and Asset Hub?

**Comment by prayagd** [comment]
@vadaynujra to test this feature there is channel needed between the foucoco and asset hub.

**Comment by ebma** [comment]
Since this is explicitly about the transfer of the relay chain token at least according to the first line in the description we don't need to involve AssetHub. Transferring ROC (Rococo's native token, ie. the relay chain token of Foucoco) to Foucoco and back should work. Which AFAIU is enough for this ticket? If you also want to test transfers of other tokens, this wouldn't be possible as is on Foucoco.

**Comment by prayagd** [comment]
We also want to support the assethub's but i can reduce the scope of this ticket to rollout this feature just for the the relay chain to and for transfers

**Comment by ebma** [comment]
For supporting AssetHub tokens we'd need to add some changes to the Foucoco runtime because IIRC they are not supported yet. If this is required for testing this ticket (or a follow-up ticket assuming you reduce the scope of this one), please create a new ticket on the Pendulum repository and list down which AssetHub assets you'd like to have available on Foucoco.

**Comment by prayagd** [comment]
Reduce the scope to only Relay chains based on the above comments, will create a follow up ticket for other channels.
